packages/expo-cli/src/commands/apply/configureIOSProjectAsync.ts

```diff
@@ -0,0 +1,90 @@
+import { IosPlist } from '@expo/xdl';
+import { IOSConfig, getConfig } from '@expo/config';
+import path from 'path';
+
+// TODO: come up with a better solution for using app.json expo.name in various places
+function sanitizedName(name: string) {
+  return name
+    .replace(/[\W_]+/g, '')
+    .normalize('NFD')
+    .replace(/[\u0300-\u036f]/g, '');
+}
+
+// TODO: it's silly and kind of fragile that we look at app config to determine the
+// ios project paths
+// * Overall this function needs to be revamped, just a placeholder for now!
+function getIOSPaths(projectRoot: string) {
+  const { exp } = getConfig(projectRoot, { skipSDKVersionRequirement: true });
+
+  let projectName = exp.name;
+  if (!projectName) {
+    throw new Error('Need a name ;O');
+  }
+
+  const iosProjectDirectory = path.join(projectRoot, 'ios', sanitizedName(projectName));
```

Comment:
Maybe we can just look up the xcodeproj with `glob`, something like this (from #1752):
```ts
const reactNativeIos = sync(path.join(projectRoot, 'ios', '*', 'Info.plist'));
path.join(reactNativeIos[0], '..');
```

Probably want to check if the `AppDelegate.m` exists in the same folder, otherwise glob sync may resolve the Test folder in iOS.